src/applications/vaos/appointment-list/sass/styles.scss

```
@@ -153,42 +156,89 @@ li.vaos-appts__listItem--clickable[data-status="pending"]:last-child {
   border-color: $color-gray-medium !important;
 }
 
-.vaos-appts__column--1 {
-  min-width: 63px;
-  max-width: 63px;
+$gap-values: (
+  0: initial,
+  1: 8px,
+  2: 16px,
+  3: 24px,
+  4: 32px,
+  5: 40px,
+);
 
-  @include media($xsmall-screen) {
-    min-width: 25px;
+// Standard utilitiles
+@each $name, $value in $gap-values {
+  .vaos-appts__column-gap--#{$name} {
+    column-gap: $value !important;
   }
-  @include media($small-screen) {
-    min-width: 63px !important;
-    max-width: 63px !important;
+}
+// Responsive Utilities
+@each $bp_name, $bp_value in $breakpoints {
+  @media (min-width: $bp_value) {
+    @each $name, $value in $gap-values {
+      .#{$bp_name}\:vaos-appts__column-gap--#{$name} {
+        column-gap: $value !important;
+      }
+    }
   }
 }
-
-.vaos-appts__column--2 {
-  @media (min-width: $small-screen) {
-    // margin-left: 95px !important;
+.vaos-appts__listItem--lineHeight {
+  @include media($small-desktop-screen) {
+    line-height: 3.5;
   }
 }
+.medium-screen\:vaos-appts__minWidth,
+.medium-screen\:vaos-appts__maxWidth {
+  min-width: 24;
+}
+.vaos-appts__column--alignItems {
+  @include media($medium-screen) {
+    align-items: stretch;
+  }
+  @include media($small-desktop-screen) {
+    align-items: center;
+  }
+}
+.vaos-appts__column--1 {
+  //
+}
 
 div[id^="vaos-appts__detail-"] {
-  @include media($medium-screen) {
+  @include media($small-screen) {
     display: flex;
-    justify-content: right;
     align-items: center;
     text-align: right;
+    min-width: fit-content;
+    max-width: fit-content;
+  }
+  @include media($small-desktop-screen) {
+    justify-content: right;
+  }
+}
+
+.vaos-appts__display--table {
+  @include media($small-screen) {
+    display: table;
+    table-layout: fixed;
+    width: 100%;
+    max-width: initial;
   }
```

Removing the `@include media` but keeping its content will apply the styling to all screen sizes. 
```suggestion
.vaos-appts__display--table {
    display: table;
    table-layout: fixed;
    width: 100%;
    max-width: initial;
```

---

Done.